src/core/agent/services/keriaNotificationService.ts

```diff
@@ -906,6 +906,11 @@ class KeriaNotificationService extends AgentService {
         //   .catch(() => undefined);
         //if (connectionRecord && !existingConnection) {
         if (connectionRecord) {
+          if (connectionRecord.pendingDeletion) {
```

Comment:
We should just do `if (connectionRecord && !connectionRecord.pendingDeletion)` to not duplicate this code, so it just go to the `else` block if pendingDeletion is true